### ISSUE TITLE
about adding chinese language support

### ISSUE THREAD

**Issue by MemphyKitay** [EDCD/EDDI#2382]
There was a contributor who provided Chinese localization for EDDI many years ago. But no one has updated since then.
These days I added Chinese resources to all the multilingual parts of the eddi project locally, and the Personality has also been initially localized (which was my original goal, before finding out that a lot of voice script were  called from within the program).

## What I'd like to happen 
Hope the EDDI group contributors can help me get these RESX into the next beta update so I can test it out. 
Or can you help me compile the Chinese version and send it to me?
## How it can happen
In the attachment I added all the Chinese resource files needed by the multilingual part right there, where they are supposed to be and attached to the project. However i dont know how to make project to complie into programm

## EDDI Version
4.0.1
[Chinese resourses for EDDI-Release-4.0.1 in position.zip](https://github.com/EDCD/EDDI/files/9534771/Chinese.resourses.for.EDDI-Release-4.0.1.in.position.zip)

**Comment by Tkael** [comment]
Hi @MemphyKitay , the typical process for accepting localizations for EDDI is documented at https://github.com/EDCD/EDDI/blob/ac863458a504d281ed4997a0b308156f8b6cac93/docs/Localization_for_translators.md. There is a lead translator for each language but if that lead translator is not responsive then we can reassign the role at our discretion.

I appreciate the work you've put into a Chinese translation. There is a way to incorporate and upload the files you've provided but we would forgo process that would normally occur via CrowdIn (where the lead translator would typically ratify the purposes translation before incorporation) so I wouldn't want to do that without the approval of the lead translator.

So TLDR, I'd like to have the original Chinese lead translator weigh in if they are available. If they are not available then would you be willing to take on that lead translator role?

o7

**Comment by MemphyKitay** [comment]
Hi Brian!
Thank you very much for your reply!
 I am more than happy to contribute to the localization of EDDI and may
remain active for a longer period of time.
 I don't know what the job as a lead translator is, but I'm honored to
temporary take that position if there is a need.


o7

Brian Wilson ***@***.***> 于 2022年9月11日周日 上午1:08写道：

**Comment by Tkael** [comment]
Ok. I think I misremembered, we don't currently have a Chinese lead translator (Proofreader).
I've checked a number of the translations and they look good, so if you are willing to work on the translation then I will grant you the role of lead translator (Proofreader).

It looks like you've submitted files using the `zh-CHS` language code. For the files you've already translated, we'll need to patch them in on this side before they will be editable via CrowdIn. We will also need to add Chinese to the list of languages on CrowdIn (I'll need to review how to do that as @VerticalBlank has typically added new languages in the past).

**Comment by richardbuckle** [comment]
This is now integrated on CrowdIn.
